docker-compose.yml

```diff
@@ -1,12 +0,0 @@
-version: '2'
-services:
-  web:
-    build: .
-    command: python manage.py dns_server
-    environment:
-        - PYTHONUNBUFFERED=1
-    volumes:
-      - .:/code
-    ports:
-      - "53:10053"
```

Comment:
@rays Actually if we're putting the dns and proxy within the same container, the dns part at least has to be on a port other than 53. This is because when the proxy is making a request, it will go through DNS resolution against the container's port 53 (which will point back to the proxy) and this causes an infinite loop.